client/src/plugins/zeebe-plugin/start-instance-plugin/StartInstanceConfigOverlay.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import React from 'react';
+
+import css from './StartInstanceConfigOverlay.less';
+
+import {
+  Overlay,
+  TextInput,
+  Section
+} from '../../../shared/ui';
+
+import {
+  Field,
+  Formik
+} from 'formik';
+
+export default class StartInstanceConfigOverlay extends React.PureComponent {
+
+   onClose = (action = 'cancel', data) => this.props.onClose(action, data);
+
+   onCancel = () => this.onClose('cancel', null);
+
+   onSubmit = (values) => this.onClose('start', values);
+
+
+   render() {
+
+     const {
+       onClose,
+       onSubmit
+     } = this;
+
+     const {
+       configuration: values,
+       anchor,
+       ref, height
+     } = this.props;
+
+     return (
+       <Overlay className={ css.StartInstanceDetailsOverlay } onClose={ onClose } anchor={ anchor } ref={ ref }>
+         <Formik
+           initialValues={ values }
+           onSubmit={ onSubmit }
+         >
+           { form => (
+             <form onSubmit={ form.handleSubmit } style={ { height: height } }>
+
+               <Section>
+                 <Section.Header>
+                   {
+                     'Start Process Instance'
+                   }
+                 </Section.Header>
+
+                 <Section.Body>
+
+                   <p className="intro">
+                     Enter details to start a process instance on Camunda Cloud.
+                   </p>
+
+                   <fieldset>
+                     <div className="fields">
+                       <Field
+                         name="variables"
+                         component={ TextInput }
+                         multiline={ true }
+                         label="Variables (optional)"
+                         description={ <p>Must be a proper JSON variables object. <a href="https://docs.camunda.org/manual/latest/reference/rest/process-definition/post-start-process-instance/#request">Learn more and find an example for variables</a>.</p> }
```

Comment:
This links to platform docs, we need to link to Cloud docs.

I propose (feel free to discuss):

```suggestion
description={ <p>Must be a proper <a href="https://www.w3schools.com/js/js_json_intro.asp">JSON string</a> representing <a href="https://docs.camunda.io/docs/components/concepts/variables/">Zeebe variables</a>.</p> }
```